src/librbd/Journal.h

```diff
@@ -11,6 +11,7 @@
 #include "common/Cond.h"
 #include "common/Mutex.h"
 #include "common/Cond.h"
+#include "common/WorkQueue.h"
```

Comment:
ahhh... I misread @trociny 's update and thought he was asking you to *add* a forward declaration for some reason (re-reading it I have no idea why). Sorry for the noise.